packages/blade/src/components/Box/LayoutPrimitivesDocs.tsx

```diff
@@ -4,7 +4,11 @@ import BaseBox from './BaseBox';
 import { Box } from '.';
 import StoryPageWrapper from '~utils/storybook/StoryPageWrapper';
 import { Code, Heading, Text, Title } from '~components/Typography';
-import { Sandbox, SandboxProvider, SandboxHighlighter } from '~utils/storybook/Sandbox';
+import {
+  Sandbox,
+  SandboxProvider,
+  SandboxHighlighter,
+} from '~utils/storybook/Sandbox/SandpackEditor';
```

Comment:
There is syntax highlighter and differently layout editor we're using from Sandpack right now. We can migrate them to stackblitz with different layout but I am scoping this task down a bit to move to rebranding.

Will fix these instances after rebranding and remove Sandpack.

<img width="1081" alt="image" src="https://github.com/razorpay/blade/assets/30949385/f0acd5fc-9f58-49ab-9291-52e7464fe88e">